sdk/communication/azure-communication-phonenumbers/src/main/java/com/azure/communication/phonenumbers/siprouting/SipRoutingAsyncClient.java

```diff
@@ -96,11 +103,12 @@ public Mono<SipTrunk> getTrunk(String fqdn) {
      * <!-- end com.azure.communication.phonenumbers.siprouting.asyncclient.getTrunkWithResponse -->
      *
      * @param fqdn SIP Trunk FQDN.
+     * @param expand Sip configuration expand. Optional.
      * @return Response object with the SIP Trunk if exists, with null otherwise.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public Mono<Response<SipTrunk>> getTrunkWithResponse(String fqdn) {
-        return getSipConfigurationWithResponse()
+    public Mono<Response<SipTrunk>> getTrunkWithResponse(String fqdn, ExpandEnum expand) {
```

Comment:
That's good, but we need the expand also on listTrunks (similar comment as in Python SDK).